### ISSUE TITLE
Future of iampicard? 

### ISSUE THREAD

**Issue by WeAreKlingons** [IAmPicard/StarTrekTimelinesSpreadsheet#271]
DB has decided to remove the iampicard thread on their forums with this text: "Hi all,

The thread related to the iampicard tool has been removed while we assess more thoroughly the ins and outs of said tool and its impacts."
should we assume that this tool will be banned and we should immediately stop using it? I really hope not because I really liked the voyage crew selection feature of the tool. Thank you for creating this tool. I just hope that it wasn't wasted time.

**Comment by IAmPicard** [comment]
I'm going to paste what I replied to a similar question on Reddit:

I received no communication from anyone at DB / TP on the matter. If they do reach out, I'll be fully transparent with the community. From my side, I did everything reasonable to adhere to their ToS (including asking for and receiving approval for contested features); I also continuously tried to open a direct line of communication for the same purpose, but unfortunately never got through.

If they request it taken down I will comply, because it's not worth the hassle for me. I already accomplished the goals I had with this project (easier crew management, and learn a bit of JS / web / server stuff) - it's now become a time sink (2nd job, "pro bono" 🙂); I made the tool open source hoping to get more contributors (you'd think a Star Trek game has a lot of users into tech), but very few actually submitted PRs on GitHub.

-- IAmPicard 🖖
